superdesk/vocabularies/__init__.py

```diff
@@ -14,7 +14,7 @@
 from .vocabularies import VocabulariesResource, VocabulariesService, is_related_content
 from .commands import UpdateVocabulariesInItemsCommand  # noqa
 from .keywords import add_missing_keywords
-from flask_babel import _
+from flask_babel import _, lazy_gettext
```

Comment:
do we need `_` here, if we are replacing it? I am just asking.

Reply:
yep it's used in the file for some error messages, we don't need to replace it everywhere

Reply:
Okay.